cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.h

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <CoreMinimal.h>
+#include <GameFramework/Pawn.h>
+
+#include "UrdfBotPawn.generated.h"
+
+class UCameraComponent;
+class UUrdfRobotComponent;
+
+// UrdfBot agent
```

Comment:
comment not needed (and is technically incorrect because "agent" has a specific technical meaning in SPEAR, referring to a higher-level object than a Pawn)